build-data-capturing-gradle-samples/capture-test-pts-support/gradle-test-pts-support.gradle.kts

```diff
@@ -85,4 +88,18 @@ class Capture(val logger: Logger) {
                 }
         }
     }
+
+    private val filterIncompatibleKotestVersions = Predicate<File> { jar ->
+        if (jar.name.contains("kotest-runner")) {
+            val kotestVersionString = jar.name.split("-")[jar.name.split("-").lastIndex].replace(".jar", "")
+            val kotestVersion = VersionNumber.parse(kotestVersionString)
+            if (VersionNumber.UNKNOWN == kotestVersion) {
+                logger.warn("Unable to parse kotest version from file name ${jar.name}")
+                return@Predicate false
```

Comment:
This looks good, I would just make both return statements consistent. Either they both have the `return@Predicate` or neither.